stm32-modules/heater-shaker/include/heater-shaker/gcodes.hpp

```diff
@@ -21,14 +21,15 @@
 
 namespace gcode {
 
+
 struct SetRPM {
     /*
     ** Set RPM uses the spindle-speed code from standard gcode, M3 (CW)
     ** Format: M3 S<RPM>
     ** Example: M3 S500 sets target rpm to 500
     */
     using ParseResult = std::optional<SetRPM>;
-    uint32_t rpm;
+    int16_t rpm;
```

Comment:
Why did we change the type of rpm here?

Reply:
To match the mc api's underlying data type. Since that's an int16_t, any extra data would just get thrown away.